README.md

```diff
@@ -3,111 +3,60 @@
-- [DualSPHysics](https://github.com/DualSPHysics/DualSPHysics)
-- [OpenFOAM](https://www.openfoam.com/)
-- [SWASH](https://swash.sourceforge.io/)
-- [xBeach](https://oss.deltares.nl/web/xbeach/)
-- [GROMACS](https://www.gromacs.org/)
+## Scenarios
 
-If you would like other simulators to be added, contact us at [simulations@inductiva.ai](mailto:simulations@inductiva.ai).
+**Inductiva API** contains pre-built scenarios that define a particular physical system ready to simulate. Users can select some parameters within the scenario that configure the system, run the simulation and visualize the results.
 
-## Scenarios
+Let's go through the currently available scenarios in **Inductiva API**.
 
-### Fluid block
+### ProteinSolvation
 
-This scenario simulates the motion of a fluid block in a unit cubic tank under
-the action of gravity. The fluid motion is simulated using the [Smoothed
-Particle Hydrodynamics](https://en.wikipedia.org/wiki/Smoothed-particle_hydrodynamics)
-method. The fluid block is initialized with a given velocity, dimensions and
-physical properties (density and viscosity).
+The ProteinSolvation scenario models the dynamics of a protein whose structure is described by a PDB file. The protein is placed in a cubic box filled with water. If the protein has a non-zero electric charge, charged ions are added to the solution to neutralize the system. First, the system undergoes an [energy minimization](https://manual.gromacs.org/current/reference-manual/algorithms/energy-minimization.html) process to eliminate any steric clashes or structural issues within the protein-water system. After this, the position of the atoms in this system is updated according to Newton's equation in discrete time steps. The force that acts upon the particles is computed using standard molecular force fields.
 
 #### Example
 
 Initialize the scenario:
+```
+from inductiva import molecules
 
-```python
-from inductiva import fluids
-
-scenario = fluids.scenarios.FluidBlock(density=1e3,
-                             kinematic_viscosity=1e-6,
-                             position=(0.3, 0.3, 0.3),
-                             dimensions=(0.4, 0.4, 0.4),
-                             initial_velocity=(0.0, 0.0, 0.0))
+scenario = molecules.scenarios.ProteinSolvation(pdb_file, temperature = 300)
 ```
 
-The user can specify the fluid density (in kg/m^3), kinematic viscosity (in
-m^2/s), initial position (in m), dimensions (in m) and velocity (in m/s).
+The user must provide the path for the PDB file (pdb_file) corresponding to the protein to be simulated. Additionally, the temperature (in Kelvin) can be specified, which defaults to 300 K. 
 
 Run the simulation:
 
 ```python
-output = scenario.simulate(simulator=fluids.SPlisHSPlasH(),
-                           simulation_time=1,
-                           time_step=0.001,
-                           output_time_step=0.05)
+output = scenario.simulate(simulation_time = 10, nsteps_minim = 5000)
```

Comment:
Remove space before and after the `=` signs for consistency with the syntax used in the scenarios below (which is standard).